domain/resourcegroup/runaway.go

```diff
@@ -252,6 +253,7 @@ func (rm *RunawayManager) DeriveChecker(resourceGroupName, originalSQL, sqlDiges
 	if group.RunawaySettings == nil && rm.activeGroup[resourceGroupName] == 0 {
 		return nil
 	}
+	metrics.RunawayCheckerCounter.WithLabelValues(resourceGroupName, "hit", "").Inc()
```

Comment:
It's better to remove `WithLabelValues`. in most cases, the group name is not changed.